core/templates/dev/head/css/oppia.css

```diff
@@ -2513,6 +2513,10 @@ md-card.preview-conversation-skin-supplemental-card {
   color: blue;
 }
 
+.oppia-info-card-content .tooltip-inner {
```

Comment:
Could you please add a comment above this line:

    /* This is necessary so that long usernames don't overflow the tooltip. */

Thanks!